action.yml

```diff
@@ -0,0 +1,202 @@
+---
+name: Steampipe Check
+author: Turbot
+description: Run Steampipe mod controls and create annotations for Infrastructure as Code controls.
+branding:
+  color: red
+  icon: shield
+inputs:
+  mod-url:
+    description: "Mod URL to clone. Examples: https://github.com/turbot/steampipe-mod-aws-compliance"
+    required: true
+  mod-branch:
+    description: "Mod branch or tag to clone. Defaults to 'main'. Examples: main, release/v0.7, 0.5, 1.0."
+    required: false
+    default: main
+  checks:
+    description: Space separated list of benchmarks and controls to run. Can also pass in as a multi-line string. Defaults to 'all'.
+    required: false
+    default: all
+  artifact-exports:
+    description: "Comma separated export types to save as artifacts. Example: 'json,csv'. Defaults to 'csv'. Valid values: 'asff,csv,html,json,md,nunit13,sps'."
+    required: false
+    default: csv
+  additional-args:
+    description: Space separated args to add to the 'steampipe check' command.
+    required: false
+  snapshot-visibility:
+    description: If set to 'anyone_with_link' or 'workspace' will create a snapshot on Turbot Pipes.
+    required: false
+  pipes-token:
+    description: The Turbot Pipes token used to save snapshots, can also be set via environment variable (https://steampipe.io/docs/reference/env-vars/pipes_token).
+    required: false
+  github-token:
+    description: GitHub token used for git/GitHub operations.
+    required: false
+    default: ${{ github.token }}
```

Comment:
To make this GHE compatible, should we use a default like https://github.com/actions/setup-node/blob/5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d/action.yml#L23C5-L23C84?

Reply:
Not sure on this one actually, as in example above has potential for a token of empty string if used from GHES & not provided, leaving as-is will at least have the token from run context in GHES which may be sufficient.

Reply:
Agreed to leave as-is